h2/src/main/org/h2/fulltext/FullTextLucene.java

```diff
@@ -58,6 +57,8 @@
      */
     protected static final boolean STORE_DOCUMENT_TEXT_IN_INDEX =
             Utils.getProperty("h2.storeDocumentTextInIndex", false);
+    protected static final String LUCENE_ANALYZER =
+            Utils.getProperty("h2.fullText.luceneAnalyzer", "org.apache.lucene.analysis.standard.StandardAnalyzer");
```

Comment:
I think that `fullText` part in not needed here. Also add a Javadoc with some description.